docs/source/chainerx/contribution.rst

```diff
@@ -112,14 +112,14 @@ Test coverage
 -------------
 
 We use `gcov <https://gcc.gnu.org/onlinedocs/gcc/Gcov.html>`_ to the measure C++ code coverage.
-Build the Python package in ``Debug`` mode, and build C++ test suite as:
+From the root of the project, build the Python package in ``Debug`` mode, and build C++ test suite as:
 
 .. code-block:: console
 
     $ python setup.py build --debug --build-temp ./build --build-lib ./build develop
     $ mkdir -p build
     $ cd build
-    $ cmake -DCMAKE_BUILD_TYPE=Debug -DCHAINERX_BUILD_PYTHON=1 -DCHAINERX_ENABLE_COVERAGE ..
+    $ cmake -DCMAKE_BUILD_TYPE=Debug -DCHAINERX_BUILD_PYTHON=1 -DCHAINERX_ENABLE_COVERAGE ../chainerx_cc
```

Comment:
This is an error that already existed before this PR but I think `-DCHAINERX_ENABLE_COVERAGE` needs a value. Could you fix that too? If not, we can address it in a separate PR.
```suggestion
    $ cmake -DCMAKE_BUILD_TYPE=Debug -DCHAINERX_BUILD_PYTHON=1 -DCHAINERX_ENABLE_COVERAGE=1 ../chainerx_cc
```

Reply:
Sure, I'll update that line and ensure there aren't any issues with the build and coverage checks.